docs/settings/cases-settings.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[cases-settings]]
+== Cases settings in {kib}
+++++
+<titleabbrev>Cases settings</titleabbrev>
+++++
+
+You do not need to configure any additional settings to use <<cases,cases>> in {kib}.
+To provide greater control over case features, you can configure the following settings in the `kibana.yml` file:
+
+`xpack.cases.files.allowedMimeTypes`::
+The MIME types that you can attach to a case, represented in an array of strings.
+The default MIME types are specified in https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/cases/common/constants/mime_types.ts[mime_types.ts].
+
+`xpack.cases.files.maxSize`::
+The file size limit for files that you can attach to a case, represented as the number of bytes. Defaults to 10 MiB for images and 100 MiB for all other MIME types.
```

Comment:
It is unclear to me how users can specify a single number that results in two different values for images and other types. Do you specify the smaller (image) limit and the other is automatically a multiple of ten?

> The file size limit for files that you can attach to a case, represented as the number of bytes. Defaults to 10 MiB for images and 100 MiB for all other MIME types.

Reply:
Yeah this is confusing. If the user does not set this configuration, we use two numbers internally. We set the maximum 
file size for images to 10 MiB. For all other mime types we set the maximum size to 100 MiB.

If the user sets a value here, we use the value for all mime types. Maybe we shouldn't talk about the image file size here because it's confusing. Instead we could put that information in some docs related to the file attachment feature. What do you think?